bin/oneoff/backfill_data/storage_apps_project_type_and_standalone

```diff
@@ -56,31 +56,44 @@ def main
   start_index = ARGV[0].to_i
   end_index = ARGV[1].to_i
   batch_size = ARGV[2].to_i
+  current_index = -1
 
   until start_index > end_index
     puts "PROCESSING: #{start_index} to #{start_index + batch_size}..."
 
     # Build the set of updates for this batch.
     updates = []
     DB[:storage_apps].where(id: start_index...(start_index + batch_size)).each do |storage_app|
-      value_parsed = JSON.parse storage_app[:value]
+      begin
```

Comment:
The "Bad JSONs in storage_apps..." email that I sent you contains a subset of the bad IDs (there are a lot of them). More generally, iterating over the rows, trying to parse the JSON, and rescuing the ensuing exceptions (with appropriate logging) should identify all bad rows.